src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -184,6 +198,9 @@ public static ConfigDef configDef() {
         return CONFIG_DEFINITION.configDef();
     }
 
+    private String _username;
+    private String _password;
```

Comment:
How about `usernameFromAuthFile` and `passwordFromAuthFile`. I don't think underscore prefix is a common Java idiom.

Reply:
I can rename the variable names.  Some java packages (and some companies) do use _ as the prefix for instance variables.